sync-version/action.yaml

```diff
@@ -36,11 +62,21 @@ runs:
     - name: Commit changes
       if: steps.git-check.outputs.modified == 'true'
       shell: bash
+      with:
+        yaml_path: ${{ steps.yaml-path.outputs.yaml_path }}
+        prev_version: ${{ steps.yaml-path.outputs.prev_version }}
       env:
         SNAP_NAME: ${{ github.event.repository.name }}
```

Comment:
Do we also need the `snap-name` input here as with other workflows?

Reply:
yup.. missed it.. Thanks!